src/main/java/codeu/model/data/Activity.java

```diff
@@ -0,0 +1,94 @@
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+import java.util.Comparator;
+
+
+/**
+ * Class representing an actvitiy.
+ * 
+ * Attributes:
+ * - ActivityType (ActivityType)
+ *  - UserRegistered
+ *  - ConversationCreated
+ *  - MessageSent
+ * - id (UUID)
+ * - creationTime (Instant)
+ */
+public class Activity implements Comparable<Activity> {
+    public enum ActivityType {
+        UserRegistered {
+            @Override
+            public String getDisplayStringFormat() {
+                // First %1$s = time
```

Comment:
since you are using the numbers in the format you can probably get rid of the the "First" "Second" text.